challengeMk2/ViewModels/SystemDetailViewModel.cs

```diff
@@ -63,17 +63,17 @@ public SystemDetailViewModel(StarSystem selectedSystem = null)
             Title = "Star System Details";
```

Comment:
The constructor should be placed before the public properties in the file.

Reply:
Sorry, it is my fault, this class should be deleted. 
It is replaced by SystemDetailCarouselViewModel.cs and not be used anymore.